crypto/hpke/hpke.c

```diff
@@ -1,7 +1,8 @@
 /*
- * Copyright 2022-2023 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright 2000-2023 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright Siemens AG 2018-2020
```

Comment:
Why is there a Siemens copyright on these files?
The date should also be changed back to 2022-2023..

Reply:
Done. Addressed.

It was copy paste mistake. Thanks!